pkg/api/common_test.go

```diff
@@ -20,6 +20,7 @@ import (
 	"github.com/grafana/grafana/pkg/infra/usagestats"
 	"github.com/grafana/grafana/pkg/models"
 	"github.com/grafana/grafana/pkg/services/accesscontrol"
+	acdb "github.com/grafana/grafana/pkg/services/accesscontrol/database"
```

Comment:
nit: we can maybe call this `accesscontroldb` instead to make it a bit more readable and consistent with how we import `accesscontrolmock` dependency?